assets/js/wcmp-frontend.js

```diff
@@ -142,9 +142,7 @@ jQuery(function($) {
         },
 
         show_delivery_options: function() {
-            if (MyParcel_Frontend.is_updated()) {
```

Comment:
Volgens mij was het probleem wat `MyParcel_Frontend.is_updated` opgelost, dat showAllDeliveryOptions niet vaak runt en alleen als het nodig is. Heb je hier een console.log geplaatst en gekeken of (na een beetje heen en weer klikken) dit niet 10 op 1 moment wordt aangeroepen?

Reply:
Als het niet werkt als de country is aangepast, moet is_updated() ook controleren op country

Reply:
Meerdere testen gedaan en alles werkt nu zoals hij zou moeten werken.